src/main/resources/avro/alleleAnnotationmethods.avdl

```diff
@@ -76,8 +76,8 @@ record SearchVariantAnnotationsRequest {
 //  union { null, array<string> } featureIds = null;
 
   /**
-  Only return variant annotations including these effects (SO terms).
-  If null, return all variant annotations.
+  Only return variant annotations including these effects (SO terms). Exact
+  matching is required. If null, return all variant annotations.
```

Comment:
This still seems a bit ambiguous to me. Maybe further clarify that an `OntologyTerm` matches if and only if all fields of the `OntologyTerm` match exactly (at least, that's my interpretation of what this means).